clients/spark-3.2-extensions/src/main/scala/org/apache/spark/sql/execution/datasources/v2/AssignReferenceExec.scala

```diff
@@ -15,12 +15,8 @@
  */
 package org.apache.spark.sql.execution.datasources.v2
 
-import org.apache.spark.sql.catalyst.InternalRow
 import org.apache.spark.sql.catalyst.expressions.Attribute
 import org.apache.spark.sql.connector.catalog.CatalogPlugin
-import org.apache.spark.unsafe.types.UTF8String
-import org.projectnessie.client.api.NessieApiV1
-import org.projectnessie.model.{Branch, Tag}
```

Comment:
we need configure spotless for scala instead of manually deleting them?

Reply:
yes

Reply:
created [#3674](https://github.com/projectnessie/nessie/issues/3674) this for tracking.